### ISSUE TITLE
Can't submit even though attempt seems to be successful

### ISSUE THREAD

**Issue by IIazertyuiopII** [Codewars/codewars.com#1411]
Hello,

I cannot submit my solution to the https://www.codewars.com/kata/naive-subarray/ kata since even though the tests are passing, a submit button never appears.

I have tried logging out, reloading the page, switching to a different browser (chrome to edge), resetting the kata and switching to a different computer, but no luck.

My issue looks exactly like #893 in the sense that the output box has a red border, but all tests are okay. I didn't want to reopen #893 since it seems to be closed since 1 year and I didn't find traces of people having issues except me recently.

My account on https://www.codewars.com/users/AZERTYUIOP.

**Comment by IIazertyuiopII** [comment]
server response below, in case that's useful
{
	"type": "execution success",
	"stdout": "\n<DESCRIBE::>Tests\n\n<IT::>should_work_for_exmaple\nsolving n = 4 range = 4\n\n<PASSED::>Test Passed\n\n<COMPLETEDIN::>\n\n<IT::>should_work_for_larger_tests\nsolving n = 15 range = 66\nsolving n = 27 range = 10\n\n<PASSED::>Test Passed\n\n<COMPLETEDIN::>\n\n<IT::>shoud_work_for_random_tests_n_1000_An_10000\n<p><font color=\"blue\">n == 1000, a[n] <= 10000</font></p>\nsolving n = 1000 range = 9991\nsolving n = 1000 range = 9997\nsolving n = 1000 range = 9994\n\n<PASSED::>Test Passed\n\n<COMPLETEDIN::>\n\n<IT::>shoud_work_for_random_tests_n_20000_An_100000\n<p><font color=\"yellow\">n == 19999, a[n] <= 100000</font></p>\nsolving n = 19999 range = 99999\nsolving n = 19999 range = 99979\n\n<PASSED::>Test Passed\n\n<COMPLETEDIN::>\n\n<IT::>shoud_work_for_random_tests_n_100000_An_100000\n<p><font color=\"red\">n == 100000, a[n] <= 100000</font></p>\n<p><font color=\"red\">test data will be stronger than you expect.</font></p>\n<p><font color=\"red\">I've done that on purpose.</font></p>\n",
	"stderr": "",
	"exitCode": 137,
	"wallTime": 10598,
	"timedOut": false,
	"message": "",
	"token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJjb2RlcnVubmVyIiwiZXhwIjoxNTI0NjEwOTg0NDgwLCJpZCI6IjVhZGI4YmRjMmYxMGM2Y2RlODAwMDBmYiIsImNpcGhlcmVkIjpbInNldHVwIiwiZml4dHVyZSJdLCJjb21wbGV0ZWQiOmZhbHNlLCJ3YWxsVGltZSI6MTA1OTh9.f5RQWe53-fFJ4CJvrk5omg0vMeNFE4qP6dGZcO7fMm0",
	"result": {
		"serverError": false,
		"completed": false,
		"output": [
			{
				"t": "describe",
				"v": "Tests",
				"p": true,
				"items": [
					{
						"t": "it",
						"v": "should_work_for_exmaple",
						"p": true,
						"items": [
							{
								"t": "log",
								"v": "solving n = 4 range = 4\n"
							},
							{
								"t": "passed",
								"v": "Test Passed"
							}
						]
					},
					{
						"t": "it",
						"v": "should_work_for_larger_tests",
						"p": true,
						"items": [
							{
								"t": "log",
								"v": "solving n = 15 range = 66\nsolving n = 27 range = 10\n"
							},
							{
								"t": "passed",
								"v": "Test Passed"
							}
						]
					},
					{
						"t": "it",
						"v": "shoud_work_for_random_tests_n_1000_An_10000",
						"p": true,
						"items": [
							{
								"t": "log",
								"v": "<p><font color=\"blue\">n == 1000, a[n] <= 10000</font></p>\nsolving n = 1000 range = 9991\nsolving n = 1000 range = 9997\nsolving n = 1000 range = 9994\n"
							},
							{
								"t": "passed",
								"v": "Test Passed"
							}
						]
					},
					{
						"t": "it",
						"v": "shoud_work_for_random_tests_n_20000_An_100000",
						"p": true,
						"items": [
							{
								"t": "log",
								"v": "<p><font color=\"yellow\">n == 19999, a[n] <= 100000</font></p>\nsolving n = 19999 range = 99999\nsolving n = 19999 range = 99979\n"
							},
							{
								"t": "passed",
								"v": "Test Passed"
							}
						]
					},
					{
						"t": "it",
						"v": "shoud_work_for_random_tests_n_100000_An_100000",
						"p": true,
						"items": [
							{
								"t": "log",
								"v": "<p><font color=\"red\">n == 100000, a[n] <= 100000</font></p>\n<p><font color=\"red\">test data will be stronger than you expect.</font></p>\n<p><font color=\"red\">I've done that on purpose.</font></p>\n"
							}
						]
					}
				]
			}
		],
		"successMode": "assertions",
		"passed": 4,
		"failed": 0,
		"errors": 0,
		"error": null,
		"assertions": {
			"passed": 4,
			"failed": 0,
			"hidden": {
				"passed": 0,
				"failed": 0
			}
		},
		"specs": {
			"passed": 4,
			"failed": 0,
			"hidden": {
				"passed": 0,
				"failed": 0
			}
		},
		"unweighted": {
			"passed": 4,
			"failed": 0
		},
		"weighted": {
			"passed": 4,
			"failed": 0
		},
		"timedOut": false,
		"wallTime": 10598,
		"testTime": null,
		"tags": null
	}
}

**Comment by kazk** [comment]
Thanks for all the details. I really appreciate it!

Looks like the cause of this is that the exit code is non-zero (`137`). This usually means the process was killed (`128` + `9` `SIGKILL`). This can be caused by many reasons and one of them is that the process is using too much memory.
Do you think that can be the cause?

I'll look into it further and let you know if I find anything.

**Comment by IIazertyuiopII** [comment]
Good point, the code is currently leaking (didn't think that would cause issues). I will fix it and try again. Thanks for the fast feedback !

**Comment by kazk** [comment]
There's a bug in the output too.
The last test case is missing `{"t": "passed", "v": "Test Passed"}` because the process was killed, but it's still considered "passed". I'll try fixing that soon.

**Comment by Voileexperiments** [comment]
@kazk this is basically https://github.com/Codewars/codewars-runner-cli/issues/601
